rpm/stackdriver-agent.spec

```diff
@@ -110,11 +108,8 @@ Source202: stackdriver.sysconfig
 BuildRequires: perl(ExtUtils::MakeMaker)
 BuildRequires: perl(ExtUtils::Embed)
 %endif
-%if ! %{has_python36}
```

Comment:
Do we need to change anything in the sles 12 and 15 dockerfile (https://github.com/Stackdriver/agent-packaging/blob/1f1ee6b739fc50fbe27d1bfc2214f6386515b70f/build/x86_64_sles12/Dockerfile#L29) to add Python 3 as well? Or do they have python2-devel and python3-devel already?

Reply:
Great catch. Yes, those and the Amazon `Dockerfile` also needed to be modified. However, they are currently broken at HEAD, so we first need to get them working again. Sent #132 — once that's merged, I'll update these.

Reply:
Restored the build containers and rebased this PR.

Reply:
Also Debian stretch, apparently. 😦 Sent #137.